File: 프로그래머스/unrated/181900. 글자 지우기/글자 지우기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string solution(string my_string, vector<int> indices) {
    sort(indices.begin(), indices.end());
    
    for (int i = indices.size() - 1; i >= 0; i--)
        my_string.erase(my_string.begin() + indices[i]);
    
    return my_string;
} 

File: 프로그래머스/0/181900. 글자 지우기/글자 지우기.java
import java.util.Arrays;
import java.util.List;
import java.util.stream.Collectors;

class Solution {
    public String solution(String my_string, int[] indices) {
        StringBuilder answer = new StringBuilder();
        
        List<Integer> list = Arrays.stream(indices)
                .boxed()
                .collect(Collectors.toList());

        for (int i = 0; i < my_string.length(); i++) {
            if (!list.contains(i)) answer.append(my_string.charAt(i));
        }
        
        return answer.toString();
    }
}